docs/commands/preview.md

```diff
@@ -14,19 +14,19 @@ This command is for our pre-release products, currently open for early access to
 redocly preview
 redocly preview --product=revel
 redocly preview --product=reef --plan=pro
-redocly preview --product=reef --plan=pro --source-dir=./my-docs-project --port=4001
+redocly preview --product=reef --plan=pro --project-dir=./my-docs-project --port=4001
```

Comment:
Is this a breaking change? Will the old alias still work?

Reply:
Added `--source-dir` as an alias.

Reply:
Add a comment to remove it in a few month then.